py/code_intelligence/triage.ipynb

```diff
@@ -54,7 +54,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 241,
+   "execution_count": null,
```

Comment:
This file has been moved to `/Issue_Triage/notebooks` in master

Reply:
Done